renderers/config/interactive_config.go

```diff
@@ -34,6 +35,7 @@ func NewDefaultEmojiRenderingConfig() *InteractiveRendererConfig {
 		ProgressIndicatorCycleDuration: time.Second,
 		SuccessStatus:                  "✅",
 		FailureStatus:                  "❌",
+		SkippedStatus:                  "⚠️ ",
```

Comment:
Why it's not rendered on GitHub? Which one is it?

Reply:
I guess it's due to fonts used here, it's rendered fine in the terminal and GMail's message from GitHub. It's ⚠️.